detail.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function()
+{
+  $(".smallsrc").mouseenter(function()
+  {
+    var want = $(this).html();
```

Comment:
The variable could be renamed to something clearer... I'm not sure what "want" refers to. Pretty ambiguous tbh

Reply:
I like how she used $(this).html()!

Reply:
@justinwride I like being ambiguous tbh